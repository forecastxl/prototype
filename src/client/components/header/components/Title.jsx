import muiThemeable from 'material-ui/styles/muiThemeable'
import styled from 'styled-components'

export const DumbTitle = styled.h1`
  color: ${props => props.muiTheme.palette.alternateTextColor};
  font-size: 1.25rem;
  font-weight: 400;
`

export default muiThemeable()(DumbTitle)
