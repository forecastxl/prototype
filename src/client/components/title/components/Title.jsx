import styled from 'styled-components'

const Title = styled.h2`
  font-size: 2rem;
`

export default Title
