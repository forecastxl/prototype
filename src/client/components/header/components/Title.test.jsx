import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { Title } from './Title'

describe('<Title />', () => {
  it('renders correctly', () => {
    const theme = {
      palette: {
        alternateTextColor: 'blue'
      }
    }
    const wrapper = shallow(<Title muiTheme={theme} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
