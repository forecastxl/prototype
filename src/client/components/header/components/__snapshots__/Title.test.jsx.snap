// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DumbTitle /> renders correctly 1`] = `
.CZTRT {
  color: blue;
  font-size: 1.25rem;
  font-weight: 400;
}

<h1
  className="sc-bdVaJa CZTRT"
/>
`;

exports[`<Title /> renders correctly 1`] = `
.CZTRT {
  color: blue;
  font-size: 1.25rem;
  font-weight: 400;
}

<h1
  className="sc-bdVaJa CZTRT"
/>
`;
